client/translations/amneziavpn_ru.ts

```diff
@@ -2674,7 +2674,7 @@ and will not be shared or disclosed to the Amnezia or any third parties</source>
     <message>
         <location filename="../core/errorstrings.cpp" line="57"/>
         <source>This config has already been added to the application</source>
-        <translation type="unfinished"></translation>
+        <translation>Этот конфиг уже добавлен в приложение</translation>
```

Comment:
```suggestion
        <translation>Этот конфиг уже был добавлен в приложение</translation>
```
Would suggest a slightly more correct translation.

Reply:
thanks for the correction!